### ISSUE TITLE
problem on debian 11

### ISSUE THREAD

**Issue by mikysal78** [nkakouros-original/ansible-role-easyrsa#8]
I try install and block on:
```
TASK [nkakouros.easyrsa : Create CA] **********************************************************************************************************************************************************
```

in my playbook i have this:
```
  vars:
    # EasyRSA
    easyrsa_generate_dh: true
    easyrsa_replace_pki: false
    easyrsa_conf_req_country: IT
    easyrsa_conf_req_province: "MT"
    easyrsa_conf_req_city: "Matera"
    easyrsa_conf_req_org: "Ninux"
    easyrsa_conf_req_email: "openwisp.basilicata@gmail.com"
    easyrsa_conf_req_ou: "Ninux Basilicata"
    easyrsa_conf_req_cn: "basilicata.ninux.org"
    easyrsa_servers:
      - name: server
    easyrsa_clients:
      - name: dns
      - name: mikytux
    easyrsa_pki_dir: /etc/easyrsa/pki
    easyrsa_renew: "{{ easyrsa_clients }}"
```

I use with role [Stouts.openvpn](https://github.com/Stouts/Stouts.openvpn)
Help me for resolve.
Best regards.

**Comment by nkakouros** [comment]
What was the issue?

**Comment by mikysal78** [comment]
Solved with version "v3.0.6" for [OpenVPN-easy-rsa](https://github.com/OpenVPN/easy-rsa/)
